docs/en/structured-data/transformation-service.md

```diff
@@ -6,11 +6,61 @@ group: Structured Data
 priority: 1
 ---
 
-* Table of Contents
-{:toc}
-
 ## Data Transformations
 
+With job service and catalog service, Alluxio can transform a table 
+to a new table at partition level. The data of the original table is not modified,
+and the data of the new table is persisted in a new location managed by Alluxio.
+Once the transformation is done, Presto users can transparently query
+against the new data.
+
+There are two kinds of supported transformations:
+
+1. coalesce the files to a specified number of files.
+2. convert CSV files to Parquet files.
+
+The transformed data is always in Parquet format.
+
+Before running a transformation, you should first attach a database.
+The following command attaches the "default" database in Hive to Alluxio.
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table attachdb hive thrift://localhost:9083 default
+```
+
+The following command coalesce files under each partition of table "test" to one file.
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table transform default test -d "write(hive).option(hive.num.files, 1)"
+```
+
+After running the above command, you'll see output like:
+
+```console
+$ Started transformation job with job ID 1572296710137, you can monitor the status of the job with './bin/alluxio table transformStatus 1572296710137'.
```

Comment:
for `console` type code blocks, you only need to put  `$` on lines which are commands that should be typed (or copy-pasted) in by the user. Output lines don't need to start with `$`